### DISCUSSION TITLE
重现文献中液态水分子的运动

### DISCUSSION THREAD

**Discussion by McAlexDudu** [deepmodeling/deepmd-kit#890] [Q&A]
开发者您好，我阅读了《DeePMD-kit: A deep learning package for many-body potentialenergy representation and molecular dynamics》文献，想问问有没有水分子vasp方面的数据，请教一下如何重现图中的数据？
<img width="478" alt="截屏2021-07-26 10 59 10" src="https://user-images.githubusercontent.com/16278257/126927197-f1af4a1b-fd45-4593-8e92-424af742f875.png">
<img width="468" alt="截屏2021-07-26 10 59 15" src="https://user-images.githubusercontent.com/16278257/126927208-ef06e58a-e8cf-45f0-9d28-c0bc47122322.png">

**Comment by njzjz** [comment]
The data can be found at http://www.deepmd.org/database/deeppot-se-data/

Note: We use English for discussion language, as many DP users don't understand Chinese (take an example, the authors of [this paper](https://doi.org/10.1126/science.abd7716)). Using English will give convenience to other researchers who want to search the previous discussion.

**Reply by McAlexDudu** [reply]
I'm sorry for that, and thanks a lot!
